docs/source/guide/tutorials/base/how_to_train/action_classification.rst

```diff
@@ -0,0 +1,275 @@
+Action Classification model
+================================
+
+This live example shows how to easily train, validate, optimize and export classification model on the `HMDB51 <https://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/>`_.
+
+.. note::
+
+  For information on the training process, including additional parameters and modification, please refer to :doc:`./detection` documentation.
+
+  For information on deploying  the trained model, please refer to the :doc:`../deploy` documentation.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.6
+
+.. note::
+
+  This example demonstates how to work with :doc:`action classification <../../../explanation/algorithms/action/action_classification>`.
+
+
+*************************
+Setup virtual environment
+*************************
+
+To create a universal virtual environment for OTX, please follow the installation process in the :doc:`quick start guide <../../../get_started/quick_start_guide/installation>`.
+
+***************************
+Dataset preparation
+***************************
+
+According to the `documentation <https://mmaction2.readthedocs.io/en/latest/supported_datasets.html#hmdb51>`_ provided by mmaction2, ensure that the `HMDB51 <https://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/>`_ dataset is structured as follows:
+
+.. code-block::
+
+    training_extensions
+    ├── data
+    │   ├── hmdb51
+    │   │   ├── hmdb51_{train,val}_split_{1,2,3}_rawframes.txt
+    │   │   ├── hmdb51_{train,val}_split_{1,2,3}_videos.txt
+    │   │   ├── annotations
+    │   │   ├── videos
+    │   │   │   ├── brush_hair
+    │   │   │   │   ├── April_09_brush_hair_u_nm_np1_ba_goo_0.avi
+    │   │   │   ├── wave
+    │   │   │   │   ├── 20060723sfjffbartsinger_wave_f_cm_np1_ba_med_0.avi
+    │   │   ├── rawframes
+    │   │   │   ├── brush_hair
+    │   │   │   │   ├── April_09_brush_hair_u_nm_np1_ba_goo_0
+    │   │   │   │   │   ├── img_00001.jpg
+    │   │   │   │   │   ├── img_00002.jpg
+    │   │   │   │   │   ├── ...
+    │   │   │   │   │   ├── flow_x_00001.jpg
+    │   │   │   │   │   ├── flow_x_00002.jpg
+    │   │   │   │   │   ├── ...
+    │   │   │   │   │   ├── flow_y_00001.jpg
+    │   │   │   │   │   ├── flow_y_00002.jpg
+    │   │   │   ├── ...
+    │   │   │   ├── wave
+    │   │   │   │   ├── 20060723sfjffbartsinger_wave_f_cm_np1_ba_med_0
+    │   │   │   │   ├── ...
+    │   │   │   │   ├── winKen_wave_u_cm_np1_ri_bad_1
+    |
+
+Once you have the dataset structured properly, you can convert it to the `CVAT <https://www.cvat.ai/>`_ format using the following command:
+
+.. code-block::
+
+  python3 otx/algorithms/action/utils/convert_public_data_to_cvat.py \
+  --task action_classification \
+  --src_path ./data/hmdb51/rawframes \
```

Comment:
Do we have a mention that HMDB dataset will be generated in data folder? AFAIN, we don't have data folder by default

Reply:
Reflected.